packages/html/src/spreadsheet/tests/spreadsheet-filter-menu.tsx

```diff
@@ -72,7 +76,7 @@ root.render(
                                             </TreeviewItem>
                                         </Treeview>
                                     </div>
-                                    <ActionButtons alignment="stretched">
+                                    <ActionButtons>
```

Comment:
please check with the design team if the action buttons alignment should be changed here